app/assets/stylesheets/application.css

```diff
@@ -109,10 +109,23 @@ img {
 }
 
 form {
-  width: 300px;
+  width: 400px;
+  text-align: left;
```

Comment:
@mikez321 you mentioned interest in responsiveness... using percentages is a great start to making sure that the form renders more usable across different screens... so let's make a note of that for the styling phase...